scripts/devenv-builder/configure-vm.sh

```diff
@@ -253,6 +254,9 @@ if ${BUILD_AND_RUN} || ${FORCE_FIREWALL}; then
 fi
 
 if ${BUILD_AND_RUN}; then
+    # shellcheck disable=SC2046
+    "${PULL_RETRY}" $(jq -r '.images | values[]' $(rpm -ql microshift-release-info | grep "release-$(uname -m)"))
```

Comment:
Before we merge this change, I'd make it a separate option and have it as false by default

Reply:
Good idea, done.